brainframe_qt/ui/dialogs/task_configuration/core/zone.py

```diff
@@ -0,0 +1,154 @@
+from abc import ABC, abstractmethod
```

Comment:
(Comment line irrelevant, I just want threaded conversations)

**Bug**:
Before: 
![image](https://user-images.githubusercontent.com/1740412/129815094-9c63bd12-643f-4679-b2c3-83f35056f342.png)


After clicking edit on "Area 4", it only lets me create a line:
![image](https://user-images.githubusercontent.com/1740412/129815120-320b32a0-6802-4057-bd42-be7759dd2aa1.png)

Expected behavior: Allow me to create a zone (which is what it was originally)

Reply:
This should be resolved

Reply:
Okay, I tested it and it works :ok_hand: